static/js/ui_init.js

```diff
@@ -457,6 +457,11 @@ export function initialize_everything() {
 
     const user_status_params = pop_fields("user_status");
 
+    // We need to initialze it first since rest
+    // of the libraries expect elements
+    // in `#compose` to be always present int he DOM.
+    compose.initialize();
```

Comment:
I think this probably doesn't need to be the very first -- just before some other specific modules.  It's probably worth actually figuring out where.

Reply:
Okay, updated the comment here.

Reply:
I think we still want it after tippy.

Reply:
Okay, updated.